packages/bsky/src/services/feed/views.ts

```diff
@@ -246,7 +249,7 @@ export class FeedViews {
     if (
       post.author.viewer?.blockedBy ||
       post.author.viewer?.blocking ||
-      blocks[uri]?.reply
+      (replyUri !== null && blocks[replyUri]?.reply)
```

Comment:
Just want to make sure I understand the intended behavior here: replies to block-violating replies are blocked, is that right?

Reply:
so a feed view post includes post views for it's replyParent & replyRoot

this basically says, the reply violates a block, then don't hydrated in the replyParent/Root

another downside of not naming this field `replyTo` 😅